pkgs/servers/http/apache-httpd/2.4.nix

```diff
@@ -38,6 +38,7 @@ stdenv.mkDerivation rec {
 
   prePatch = ''
     sed -i config.layout -e "s|installbuilddir:.*|installbuilddir: $dev/share/build|"
+    sed -i support/apachectl.in -e 's|@LYNX_PATH@|${lynx}/bin/lynx|'
```

Comment:
Why the hack are they using an actual browser instead of just doing http?

Reply:
@Mic92 I've never seen this tool before. Looks useful.

Unfortunately in this case it looks like httpie is not capable of rendering the html:
https://github.com/jakubroztocil/httpie/issues/420

Reply:
What I mean to say is that it requires a browser engine instead of issuing http to check the status to test the server.